client/dashboard/sites/settings-sftp-ssh/enable-sftp-card.tsx

```diff
@@ -0,0 +1,121 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+	Panel,
+	PanelBody,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteSftpUsersCreateMutation } from '../../app/queries';
+import { SectionHeader } from '../../components/section-header';
+
+const FILEZILLA_URL = 'https://filezilla-project.org/';
+
+export default function EnableSftpCard( {
+	siteSlug,
+	canUseSsh,
+}: {
+	siteSlug: string;
+	canUseSsh: boolean;
+} ) {
+	const mutation = useMutation( siteSftpUsersCreateMutation( siteSlug ) );
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	const handleCreateCredentials = () => {
+		mutation.mutate( undefined, {
+			onSuccess: () => {
+				createSuccessNotice( __( 'Credentials have been successfully created.' ), {
+					type: 'snackbar',
+				} );
+			},
+			onError: () => {
+				createErrorNotice(
+					__(
+						'Sorry, we had a problem retrieving your SFTP user details. Please refresh the page and try again.'
+					),
+					{
+						type: 'snackbar',
+					}
+				);
+			},
+		} );
+	};
+
+	return (
+		<Card>
+			<CardBody>
+				<VStack style={ { paddingBottom: '12px' } }>
```

Comment:
Do we still need this even though we already have `<SectionHeader/>`? 😂

Reply:
Yes.

Reply:
In Figma, we have a [`DataFormFields`](https://www.figma.com/design/dhW7zYBamXJIeJIE3wy5f4/Automattic-Components?node-id=114-10085&t=2ptbm6hr6wGmH7rS-4) component which handles the spacing tweaks. I don't think this exists in code yet though.

Reply:
> In Figma, we have a DataFormFields component which handles the spacing tweaks. I don't think this exists in code yet though.

Yup, but I think it has a flaw. I am not sure if the header is part of "DataFormField" component; it really is the header of the card (or section) itself.